backend/src/app/modules/quiz/quiz.service.ts

```
@@ -50,9 +50,48 @@ const getQuiz = async (user: JwtPayload, quizId: string) => {
+                    module: {
+                        include: {
+                            milestone: true
+                        }
+                    }
+
+                }
             }
         }
     })
+
+    let result: any = quiz
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'result' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)